src/jit/gentree.h

```diff
@@ -4037,26 +3884,198 @@ struct GenTreeIndir : public GenTreeOp
 #endif
 };
 
-// gtObj  -- 'object' (GT_OBJ). */
+// gtBlk  -- 'block' (GT_BLK, GT_STORE_BLK).
+//
+// This is the base type for all of the nodes that represent block or struct
+// values.
+// Since it can be a store, it includes gtBlkOpKind to specify the type of
+// code generation that will be used for the block operation.
+
+struct GenTreeBlk : public GenTreeIndir
+{
+public:
+    // The data to be stored (null for GT_BLK)
+    GenTree*& Data()
```

Comment:
I am merely following the convention used elsewhere (see `Addr` in the base type - `GenTreeIndir`)

---
In reply to: [77272285](https://github.com/dotnet/coreclr/pull/7027#discussion_r77272285) [](ancestors = 77272285)